pxr/usd/usdGeom/camera.h

```diff
@@ -574,6 +711,19 @@ class UsdGeomCamera : public UsdGeomXformable
     ///
     USDGEOM_API
     void SetFromCamera(const GfCamera &camera, const UsdTimeCode &time);
+
+    /// Computes the ratio between incident luminance and photometric exposure 
+    /// (in lux-seconds), given the <tt>exposure</tt>, <tt>exposure:iso</tt>, 
+    /// <tt>exposure:fnumber</tt>, <tt>exposure:time</tt> and <tt>exposure:responsivity</tt> 
+    /// attributes.
+    ///
+    /// This is expected to be applied as a multiplier to the brightness of the 
+    /// image generated by the renderer, and given physically meaningful lighting
+    /// values in the scene, allows the exposure controls on UsdGeomCamera to behave
+    /// like those of a real camera.
+    ///
+    USDGEOM_API
+    float GetExposureScale(UsdTimeCode time=UsdTimeCode::Default()) const;
```

Comment:
@anderslanglands I'm going to change this to "Compute".  I was wondering a bit about the fact that it's linear, when at least hydra is expecting something log-scale.  Is it valuable to have a linear result, or would it be better to put the log inside the function for consistency?  (If there's a use for it in linear that would lose precision from the exp2(log2(*)) that's a compelling reason to leave it as is).

If we leave it linear, I might go a step further and call it ComputeLinearExposureScale...  Anyway, thoughts?

(Don't worry about updating this, I've pulled it and it's marked up a bunch for compatibility with internal code so it's easier for me to push the last few changes through).